hyperspy/signal.py

```diff
@@ -1951,6 +1951,8 @@ def plot(self, navigator="auto", axes_manager=None,
 
         if axes_manager is None:
             axes_manager = self.axes_manager
+        if convert_units:
```

Comment:
I don't think that it is a good idea to add this option to ``plot``: the user may not be aware that this is actually changing the AxesManager units in-place. If that's what the user wants to do, he could call ``AxesManager.concert_units`` manually before calling plot.

It would be possible to change the units only for the plot axes_manager, but that would lead to issues with the ROIs.

Reply:
Yes, it is removed now.